docs/reference/getting-started.asciidoc

```diff
@@ -254,7 +254,7 @@ If everything goes well with installation, you should see a bunch of messages th
 
 --------------------------------------------------
 
-Without going too much into detail, we can see that our node named "6-bjhwl" (which will be a different set of characters in your case) has started and elected itself as a master in a single cluster. Don't worry yet at the moment what master means. The main thing that is important here is that we have started one node within one cluster.
+Without going too much into detail, we can see that our node named "localhost.localdomain" (which will be a different set of characters in your case) has started and elected itself as a master in a single cluster. Don't worry yet at the moment what master means. The main thing that is important here is that we have started one node within one cluster.
```

Comment:
could you also remove the "(which will be a different set of characters in your case)" which is no longer true, please?